ppa/archive/models.py

```diff
@@ -121,6 +122,76 @@ def stats():
         return stats
 
 
+class ProtectedFlags(Flags):
+    '''Flag set of fields that should be protected if edited in the admin.'''
+    title = ()
+    subtitle = ()
+    sort_title = ()
+    enumcron = ()
+    author = ()
+    pub_place = ()
+    publisher = ()
+    pub_date = ()
+
+    @classmethod
+    def deconstruct(cls):
+        '''Give Django information needed to make
+        :class:`ProtectedField.no_flags` default in migration.'''
+        # (import path, [args], kwargs)
+        return ('ppa.archive.models.ProtectedFlags', ['no_flags'], {})
+
+    @classmethod
+    def all_fields(cls):
```

Comment:
I thought you should just be able to iterate over the class to get the defined fields, but I see from experimentation that it returns the verbose name from the string method! I'm not sure that's actually the behavior we want?

Reply:
I think we're done with since we're not doing verbose names.